### PR TITLE
React fron

### PR DESCRIPTION
**Pull request by dn1470**
React skeleton

### CODE REVIEW

**Comment by The-TT-Hacker** [comment]
whats this submodule?

File: .vscode/dryrun.log [hunk 1]
```diff
@@ -0,0 +1,5 @@
+make --dry-run --always-make --keep-going --print-directory
+make: Entering directory '/home/youngju/comp9447-team2'
```
Comment (The-TT-Hacker) [inline .vscode/dryrun.log:2]: should't have absolute file paths in the repo

File: .vscode/targets.log [hunk 2]
```diff
@@ -0,0 +1,286 @@
+make all --print-data-base --no-builtin-variables --no-builtin-rules --question
```
Comment (The-TT-Hacker) [inline .vscode/targets.log:1]: what are all these vscode things?

**Comment by dn1470** [comment]
It's just me and Rubait to add react pages. I didn't want to replace the current frontend directory, so just added the new react app as a sub directory

**Comment by thefishua** [comment]
Can't actually see whats in oauth react

**Comment by The-TT-Hacker** [comment]
you should be using the current frontend and extending it

**Pull request closed without merging**